uber/models.py

```diff
@@ -856,6 +862,7 @@ def model_mixin(cls, model):
 
 
 class Group(MagModel, TakesPaymentMixin):
+    public_id    = Column(UUID, default=lambda: str(uuid4()))
```

Comment:
Ok, so the goal here is to make a second UUID rather than fix the fundamental problem that we trust the primary key with full database access? Again, why not just fix that the system blindly trusts UUIDs at all? This feels to me like adding another layer of duct tape instead of fixing the design at its actual weak point.